examples/shopping/carts.rst

```diff
@@ -0,0 +1,107 @@
+.. _shopping-carts:
+
+Shopping Carts
+==============
+
+At this point, we have Captured Collections setup for users and products, and we're ready to
+start building the shopping carts. We'll start by defining a Captured Collection for cart
+updates, which will hold a record of each time a user adds or modifies a product in their cart.
+These cart updates will then be joined with the product information so that we'll have the price
+of each item. Then we'll create a "carts" Collection that rolls up all the joined updates into a
+single document that includes all the items in a users cart, along with their prices.
+
+Summing Item Quantities
+-----------------------
+
+Here's the Collection and schema for the cart updates:
+
+.. literalinclude:: cart.flow.yaml
+    :language: yaml
+    :lines: 5-7
```

Comment:
`:lines:`  make me a bit nervous because they _so easily_ bit rot, particularly when there's action at a distance like this. As an alternative, I've used flow.yaml's which tightly focus on the thing to show, and just present the whole file.

Reply:
Yeah, I agree, and especially given how many there are in here. I'll see if I can get them each into separate files, and at least try to minimize `:lines:` to just one or two places if it can't be eliminated entirely.